exoscale/datasource_exoscale_compute_ipaddress_test.go

```diff
@@ -0,0 +1,99 @@
+package exoscale
+
+import (
+	"errors"
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccDatasourceComputeIPAddress(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(`
+%s
+
+data "exoscale_compute_ipaddress" "ip_address" {
+  zone = "ch-gva-2"
+}`, testAccIPAddressConfigCreate),
```

Comment:
It's not even necessary to actually create an EIP in this test step, since it's supposed to fail before even performing API calls.